spec.md

```diff
@@ -0,0 +1,238 @@
+
+Large files can benefit from compression when being transferred across the network. However, decompression takes time and compute cycles. For smaller, individual files, the decompression may take longer than downloading the file in its original format.
+
+### Layer Versioning
+
+Layers MUST be versioned to future proof new enhancements that may evolve. How the artifact tooling processes forward and backward compatibility is up to the artifact and tooling authors.
+
+### Defining Layer Types
+
+Artifact layers are expected to be unique, per artifact author. A layer that represents an OCI Image is not expected to be shared with a Helm Chart. To differentiate layers, artifact authors SHOULD create unique artifact layer types.
+
+Artifact layer types follow the same convention as manifests, utilizing the `layer.mediaType` with the following format:  
+`application/vnd.`[org|company]`.`[layerType]`.`[layerSubType]`.layer.`[version]`+`[fileFormat]
+
+### Example Layer Types
+
+|Artifact Type|mediaType|
+|-|-|
+|OCI Image|`application/vnd.oci.image.layer.v1+tar`|
+|Docker Image|`application/vnd.docker.image.rootfs.diff.tar.gzip`|
+|Helm Chart|`application/vnd.cncf.helm.chart.layer.v1+tar`|
+|Singularity SIF Layer|`application/vnd.sylabs.sif.layer.v1+tar`|
+
+### One or More Layer Types
+
+> NOTE: Should this section be in the `spec.md`, or in an `artifact-authors.md` doc?
+
+The number of layer types is up to the artifact author. Some things to consider when designing the layer format:
+
+- **Continuos Builds:** will developers automate building and pushing the artifact to a registry? For each rebuild, what will change? Consider the docker layering model, where users benefit from base layers that don't change often. If only a small piece of content changes, can you separate that content into a unique layer?
+- **Layer reuse**: when layers are pushed to a registry, layers can be de-duped. If your artifact type may benefit from shared layers across many artifact instances, consider splitting up the layers to those that change often, and those that don't.
+- **Layer sizes**: does your artifact have large content that may benefit from being split up in to smaller elements that can be concurrently uploaded and/or downloaded to a registry?
+- **Content and Config**: While a general best practice for container images says the image in the registry should not contain environmental configuration, you may be choosing to push environmental configuration to a registry as it's own artifact. If the same artifact is pushed to the same registry multiple times, only unique by some configuration, consider splitting up the configuration into its own layer, to benefit from layer reuse.
```

Comment:
nit: "it's"->"its"

Reply:
Fixed, but moving this to the [artifact authors section](https://github.com/opencontainers/artifacts/pull/6) as it's more in the opinion category.